### PR TITLE
mgr/dashboard: Update HACKING.rst

### PR DESCRIPTION
**Pull request by tspmelo**
This will update information regarding npm.

Fixes: https://tracker.ceph.com/issues/44927

Signed-off-by: Tiago Melo <tmelo@suse.com>

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/HACKING.rst [hunk 1]
```diff
@@ -220,9 +220,21 @@ Angular CLI:
 Package installation
 ~~~~~~~~~~~~~~~~~~~~
 
-Run ``npm install`` in directory ``src/pybind/mgr/dashboard/frontend`` to
+Run ``npm ci`` in directory ``src/pybind/mgr/dashboard/frontend`` to
 install the required packages locally.
 
+Adding or Updating packages
```
Comment (votdev) [inline src/pybind/mgr/dashboard/HACKING.rst:226]:
```suggestion
Adding or updating packages
```

Comment (tspmelo) [reply src/pybind/mgr/dashboard/HACKING.rst]: Done.

**Comment by tspmelo** [comment]
jenkins test make check

File: src/pybind/mgr/dashboard/HACKING.rst [hunk 2]
```diff
@@ -263,6 +275,7 @@ HTML files:
 
 - `codelyzer <http://codelyzer.com/>`_
 - `html-linter <https://github.com/chinchiheather/html-linter>`_
+- `html-linter <https://github.com/htmllint/htmllint-cli>`_
```
Comment (bk201) [inline src/pybind/mgr/dashboard/HACKING.rst:278]:
```suggestion
- `htmllint-cli <https://github.com/htmllint/htmllint-cli>`_
```

To fix the linting error.

Comment (tspmelo) [reply src/pybind/mgr/dashboard/HACKING.rst]: Done

**Comment by tspmelo** [comment]
jenkins retest this please

**Comment by tspmelo** [comment]
jenkins retest this please

**Comment by tspmelo** [comment]
jenkins retest this please

**Pull request merged**